sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -115,6 +116,8 @@ case class CreateTableLikeCommand(
       CatalogTableType.EXTERNAL
     }
 
+    val newProperties =
```

Comment:
plz leave some comments about why the metastore props are remvoed here? https://github.com/apache/spark/pull/28647#issuecomment-634373422